internal/integration/unified_spec_test.go

```diff
@@ -45,7 +45,7 @@ const (
 )
 
 var (
-	defaultHeartbeatInterval = 50 * time.Millisecond
+	defaultHeartbeatInterval = 500 * time.Millisecond
```

Comment:
What is the history of the original 50ms default? Does the 500ms requirement conflict with legacy test specifications? Or was this value chosen arbitrarily?

Reply:
Looks like it came from this [commit](https://github.com/mongodb/mongo-go-driver/commit/283326fce41bd5867f95e2638ab893db09805a8e#diff-efa6a0a1cf9acea8715073616ef3fe89c61d87dd8221555cfceb312165cf96cdR44) to implement streaming heartbeat protocol, not sure how the value was chosen though.